### DISCUSSION TITLE
How can we use `query data` in `Plotly JSON schema`?

### DISCUSSION THREAD

**Discussion by Imanedu** [ToolJet/ToolJet#6626] [Q&A] [answered]
### Environment

    Docker

### What is the expected behaviour?

   I want to use query data in Plotly JSON schema but can not any way make it work.

Is there any solution? 
Thanks

**Comment by withshubh** [comment, marked as answer]
Hello @Imanedu 👋

You can input the Plotly data as a string in the `JSON description` field. Assuming that the data returned by the query is an object, you can convert the `query data` into a string format using the `JSON.stringify()` function. 
In the JSON description field, you should include the following code:

```
{{JSON.stringify(queries.queryname.data)}}
```

Remember to replace `queryname` with the actual name of your query. This code will convert the `data` object of your query into a string and return it.

Please refer to the provided screenshot:
![Screenshot 2023-06-01 at 4 52 10 PM](https://github.com/ToolJet/ToolJet/assets/25361949/45324285-3e4d-48b0-b89b-b9a888340ad8)

**Comment by Imanedu** [comment]
Thank You 😄 👍
